DotNetClient/DotNetClient/Services/HeimdallService.cs

```diff
@@ -9,137 +9,28 @@
 
 namespace DotNetClient.Services
 {
-    public class HeimdallService
+    public partial class HeimdallService
     {
-        private readonly HttpClient heimdallClient;
-        private DateTime toDate;
-        private DateTime fromDate;
+        private readonly HttpClient _heimdallClient;
+        private readonly DateTime _toDate;
+        private readonly DateTime _fromDate;
 
```

Comment:
Det jeg mente ved const var f.eks. å ha `private readonly string _linesEndpoint = "api/v1/lines"`. For de forskjellige endepunktene, så når man skal endre de frem i tid eller gjenbruke så er de samlet